frontend/src/libs/components/prompt-generation/styles.module.scss

```diff
@@ -0,0 +1,45 @@
+.container {
+  padding: 32px 0 50px;
+}
+
+.header {
+  margin-bottom: calc(36px - 29px / 2);
+  color: var(--dark-grey);
+  font-weight: 700;
+  font-size: 24px;
+  font-family: "Libre Baskerville", serif;
+  line-height: 30px;
+}
+
+.promptsContainer {
+  display: grid;
+  grid-template-columns: repeat(6, auto);
+  gap: 20px;
+}
+
+.shuffleButton {
+  align-self: end;
+  width: 78px;
+  height: 78px;
+  margin-left: calc(31px - 20px);
```

Comment:
Do you need to use "calc" here on lines 6 and 24?

Reply:
Not really, will remove it then